src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final double radius;
+    private final String color;

```

Comment:
Please note that every figure has a color - why not put it in the base class then?